cime_config/testdefs/testlist_cam.xml

```diff
@@ -1,1108 +1,88 @@
-  </test>
-
-  <test compset="FHIST_BGC" grid="f09_f09_mg17" name="ERP_Ln9" testmods="cam/outfrq9s">
+  <test compset="F2000climo" grid="f19_f19_mtn14" name="ERP_Ln9" testmods="cam/outfrq9s">
     <machines>
-      <machine name="cheyenne" compiler="intel" category="aux_cime_baselines"/>
-      <machine name="cheyenne" compiler="intel" category="prealpha"/>
+      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
+      <machine name="fram" compiler="intel" category="aux_cam_noresm"/>
     </machines>
     <options>
       <option name="wallclock">00:10:00</option>
     </options>
   </test>
-
-  <test compset="FHIST_BDRD" grid="f09_f09_mg17" name="ERP_Ln9" testmods="cam/outfrq9s">
+  <test compset="QPC6" grid="f19_f19_mtn14" name="ERP_D_Ln9" testmods="cam/outfrq9s">
     <machines>
-      <machine name="cheyenne" compiler="intel" category="aux_cam"/>
-      <machine name="cheyenne" compiler="intel" category="aux_cam_short"/>
+      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
     </machines>
     <options>
       <option name="wallclock">00:10:00</option>
     </options>
   </test>
-
-  <test compset="F1850" grid="f09_f09_mg17" name="ERP_Ln9" testmods="cam/outfrq9s">
+  <test compset="FHS94" grid="f19_f19_mtn14" name="ERP_Ln9" testmods="cam/outfrq9s">
     <machines>
-      <machine name="cheyenne" compiler="intel" category="aux_cime_baselines"/>
-      <machine name="cheyenne" compiler="intel" category="prealpha"/>
-      <machine name="cheyenne" compiler="intel" category="aux_cam"/>
-      <machine name="cheyenne" compiler="intel" category="aux_cam_short"/>
+      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
     </machines>
-    <options>
-      <option name="wallclock">00:20:00</option>
-    </options>
   </test>
 
-<!--              @@@@@@@@@@@@@@@@@@@@@@@@@@@                 -->
-<!--              CAM test-supported COMPSETS                 -->
-<!--              @@@@@@@@@@@@@@@@@@@@@@@@@@@                 -->
-
-<!--              CAM Tests at non-scientific resolutions     -->
-  <test compset="F2000climo" grid="f19_f19_mg17" name="ERP_Ln9" testmods="cam/cosp">
-    <machines>
-      <machine name="cheyenne" compiler="intel" category="prebeta"/>
-    </machines>
-    <options>
-      <option name="wallclock">00:10:00</option>
-    </options>
-  </test>
-  <test compset="F2000climo" grid="f19_f19_mg17" name="ERP_Ln9" testmods="cam/outfrq9s">
+  <test compset="NF1850norbc" grid="f19_f19_mtn14" name="ERP_Ln9" testmods="cam/outfrq9s">
     <machines>
-      <machine name="cheyenne" compiler="gnu" category="prealpha"/>
+      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
     </machines>
     <options>
       <option name="wallclock">00:10:00</option>
     </options>
   </test>
-
-  <test compset="F2010climo" grid="f09_f09_mg17" name="SMS_Ln9" testmods="cam/nudging">
+  <test compset="NF1850norbc" grid="f19_f19_mtn14" name="SMS_D_Ln9" testmods="cam/outfrq9s">
```

Comment:
If the `ERP` test passes, what does an `SMS` test add?

Reply:
The SMS tests were debug tests as opposed to the ERP tests which were not.

Reply:
:+1: